src/primitives.jl

```diff
@@ -153,6 +153,12 @@ end
 _isnull(x::Any) = false
 _isnull(x::Nullable) = isnull(x)
 
+if VERSION < v"0.6.0-dev.2107"
```

Comment:
I know @ararslan took the time to check the commit version, but in general it's simpler and better to just do `isdefined(Base, :uniontypes)`.